### PR TITLE
[Merged by Bors] - Ipv6 bootnodes update

### PR DESCRIPTION
**Pull request by AgeManning**
We now officially have ipv6 support. The mainnet bootnodes have been updated to support ipv6. This PR updates lighthouse's internal bootnodes for mainnet to avoid fetching them on initial load.

### CODE REVIEW

**Comment by divagant-martian** [comment]
can connect to the two new bootnodes just fine :zap: 

but you forgot to update the tests :stuck_out_tongue:

**Comment by AgeManning** [comment]
Yeah, i changed that test. Not sure what its testing, but I think updating ENRs we shouldn't have to update this random test with the exact value. 

Changed the test, to just make sure the count is the same.

File: common/eth2_network_config/built_in_network_configs/mainnet/boot_enr.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
...
-- enr:-Jq4QItoFUuug_n_qbYbU0OY04-np2wT8rUCauOOXNi0H3BWbDj-zbfZb7otA7jZ6flbBpx1LNZK2TDebZ9dEKx84LYBhGV0aDKQtTA_KgEAAAD__________4JpZIJ2NIJpcISsaa0ZiXNlY3AyNTZrMaEDHAD2JKYevx89W0CcFJFiskdcEzkH_Wdv9iW42qLK79ODdWRwgiMo
-- enr:-Jq4QN_YBsUOqQsty1OGvYv48PMaiEt1AzGD1NkYQHaxZoTyVGqMYXg0K9c0LPNWC9pkXmggApp8nygYLsQwScwAgfgBhGV0aDKQtTA_KgEAAAD__________4JpZIJ2NIJpcISLosQxiXNlY3AyNTZrMaEDBJj7_dLFACaxBfaI8KZTh_SSJUjhyAyfshimvSqo22WDdWRwgiMo
+- enr:-Le4QPUXJS2BTORXxyx2Ia-9ae4YqA_JWX3ssj4E_J-3z1A-HmFGrU8BpvpqhNabayXeOZ2Nq_sbeDgtzMJpLLnXFgAChGV0aDKQtTA_KgEAAAAAIgEAAAAAAIJpZIJ2NIJpcISsaa0Zg2lwNpAkAIkHAAAAAPA8kv_-awoTiXNlY3AyNTZrMaEDHAD2JKYevx89W0CcFJFiskdcEzkH_Wdv9iW42qLK79ODdWRwgiMohHVkcDaCI4I
+- enr:-Le4QLHZDSvkLfqgEo8IWGG96h6mxwe_PsggC20CL3neLBjfXLGAQFOPSltZ7oP6ol54OvaNqO02Rnvb8YmDR274uq8ChGV0aDKQtTA_KgEAAAAAIgEAAAAAAIJpZIJ2NIJpcISLosQxg2lwNpAqAX4AAAAAAPA8kv_-ax65iXNlY3AyNTZrMaEDBJj7_dLFACaxBfaI8KZTh_SSJUjhyAyfshimvSqo22WDdWRwgiMohHVkcDaCI4I
+- enr:- Le4QH6LQrusDbAHPjU_HcKOuMeXfdEB5NJyXgHWFadfHgiySqeDyusQMvfphdYWOzuSZO9Uq2AMRJR5O4ip7OvVma8BhGV0aDKQtTA_KgEAAAAAIgEAAAAAAIJpZIJ2NIJpcISLY9ncg2lwNpAkAh8AgQIBAAAAAAAAAAmXiXNlY3AyNTZrMaECDYCZTZEksF-kmgPholqgVt8IXr-8L7Nu7YrZ7HUpgxmDdWRwgiMohHVkcDaCI4I
```
Comment (one-three-three-seven) [inline common/eth2_network_config/built_in_network_configs/mainnet/boot_enr.yaml]: There is an unnecessary space after "enr:-"

Comment (AgeManning) [reply common/eth2_network_config/built_in_network_configs/mainnet/boot_enr.yaml]: Yep, thanks for the pickup

**Review by paulhauner — APPROVED** [review]
Looks good to me, happy to merge after we have @antondlr's blessing.

**Comment by divagant-martian** [comment]
this makes me happy :smiling_face_with_tear:

**Pull request closed without merging**